src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 0;
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: all 300ms;
```

Comment:
You don;t want to use `transition: all` always write out all of the properties that will change.